release/burushos/burushos.bsk-latn.burushaski/source/burushos.bsk-latn.burushaski.model.kps

```diff
@@ -20,12 +20,12 @@
     <Name URL="">Burushaski</Name>
     <Copyright URL="">Copyright © Burusho</Copyright>
     <Author URL="">Burusho</Author>
```

Comment:
Maybe more sadness.
If the author name is "Burusho", should the parent folder change from "burushos" to "burusho"? And then changing the **other**model to match?

Reply:
I opted to match the existing model which has already been widely deployed. It's much messier to go the other way, and not worth the pain IMHO.